tests/ds_benchmark.h

```diff
@@ -289,4 +289,20 @@ static void _bench_init_perfcounters(void) {
         PRINT_TIMER_AVG(op_name)                                  \
     }
 
+#define TIME_OPERATION_SECONDS_MAXIT(op, op_name, secs, maxit, refresh)                                           \
```

Comment:
just minor typos here, I think you mean Matrix, is it right?

Reply:
No, it is really maxit for max iterations.
A stateful hash has a maximum number of signatures, which can be as low as 2^5=32, so after that many iterations it doesn't properly sign anymore.
To ensure this effect does not result in invalid benchmarks, we have to generate a fresh secret key once in a while.

To that affect I've added a macro TIME_OPERATION_SECONDS_MAXIT that uses 2 nested loops instead and has 2 additional parameters: max innerloop iterations & outerloop code.